docs/assets/extra.css

```diff
@@ -156,3 +156,8 @@ code.doc-symbol-json {
 code.doc-symbol-json::after {
   content: 'json';
 }
+
+/* Don't render view source button in mkdocstrings tooltips (see mkdocstrings/python/material/*.jinja) */
```

Comment:
"Don't render" or "Hide"?

Reply:
I applied this change in 189fcab7d634a129d31bfb5c5fd1de4e2ee3cbd9.